release-notes/opensearch-release-notes-2.9.0.md

```diff
@@ -0,0 +1,630 @@
+* Added support for editing correlation rule. ([#643](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/643))
+
+### OpenSearch Sql
+
+* Enable Table Function and PromQL function ([#1719](https://github.com/opensearch-project/sql/pull/1719))
+* Add Spark connector ([#1780](https://github.com/opensearch-project/sql/pull/1780))
+
+## ENHANCEMENTS
+
+
+### OpenSearch Anomaly Detection
+
+* Enforce DOCUMENT Replication for AD Indices and Adjust Primary Shards ([#948](https://github.com/opensearch-project/anomaly-detection/pull/948))
+
+
+
+### OpenSearch Anomaly Detection Dashboards
+* Remove invalid link ([#470](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/470))
+* Remove global OUI style override ([#511](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/511))
+* Fixing unit tests plus ran prettier ([#546](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/546))
+
+
+
+### OpenSearch ML Commons
+
+* Create model group automatically with first model version ([#1063](https://github.com/opensearch-project/ml-commons/pull/1063))
+* Init master key automatically ([#1075](https://github.com/opensearch-project/ml-commons/pull/1075)))
+
+
+### OpenSearch ML Commons Dashboards
+
+* Open external link in new blank page. (#211)
+
+### OpenSearch Performance Analyzer
+* Remove heap allocation rate as the input metric to HotShardClusterRca [#411](https://github.com/opensearch-project/performance-analyzer-rca/pull/411)
+* Set ThreadMetricsRca evaluation period from 12 seconds to 5 seconds [#410](https://github.com/opensearch-project/performance-analyzer-rca/pull/410)
+* Add unit tests for the REST layer in RCA Agent [#436](https://github.com/opensearch-project/performance-analyzer-rca/pull/436)
+
+
+
+### OpenSearch Security
+
+* Use boucycastle PEM reader instead of reg expression ([#2877](https://github.com/opensearch-project/security/pull/2877))
+* Adding field level security test cases for FlatFields ([#2876](https://github.com/opensearch-project/security/pull/2876)) ([#2893](https://github.com/opensearch-project/security/pull/2893))
+* Add password message to /dashboardsinfo endpoint ([#2949](https://github.com/opensearch-project/security/pull/2949)) ([#2955](https://github.com/opensearch-project/security/pull/2955))
+* Add .plugins-ml-connector to system index ([#2947](https://github.com/opensearch-project/security/pull/2947)) ([#2954](https://github.com/opensearch-project/security/pull/2954))
+* Parallel test jobs for CI ([#2861](https://github.com/opensearch-project/security/pull/2861)) ([#2936](https://github.com/opensearch-project/security/pull/2936))
+* Adds a check to skip serialization-deserialization if request is for same node ([#2765](https://github.com/opensearch-project/security/pull/2765)) ([#2973](https://github.com/opensearch-project/security/pull/2973))
```

Comment:
@prudhvigodithi This will have to be removed as we are reverting this PR until the potential regression is resolved.

Reply:
The tool looks for the release notes part of the plugin repo.
https://github.com/opensearch-project/security/blob/main/release-notes/opensearch-security.release-notes-2.9.0.0.md

Reply:
Those bullet points are part of the security 2.9.0 release notes. https://github.com/opensearch-project/security/blob/main/release-notes/opensearch-security.release-notes-2.9.0.0.md